server.ts

```diff
@@ -33,6 +33,31 @@ app.post('/api/graphql', async (req, res) => {
     return;
 });
 
+type Error = {
+    statusCode: number,
+    message: string,
+    __code: number
+}
+
+app.post('/api/getSignaturesForAddress', async(req, res, next) => {
+    try{
+        const { address } = req.body;
+
+        let results = await rpc.getSignaturesForAddress(address).send();
+        const signatures = results.map(result => result.signature );
+        console.log(signatures)
+        res.status(200).send(JSON.stringify(signatures));
+    }catch(e:any){
+        console.error(e)
+        let error:Error = { statusCode: 500, message: e?.context?.__serverMessage, __code: e?.context?.__code };
+        next(error);
+    }
```

Comment:
I appreciate the attempt, but since it's only a demo app, you can just log the error and send a 500 like I do with `/graphql`.

```ts
console.error(e);
res.status(500).send('Server Error');
```